ts/features/itwallet/common/store/reducers/preferences.ts

```diff
@@ -0,0 +1,28 @@
+import { addMonths } from "date-fns";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../../store/actions/types";
+import { itwCloseFeedbackBanner } from "../actions/preferences";
+
+export type ItwPreferencesState = {
+  hideFeedbackBannerUntilDate?: string;
+};
+
+const INITIAL_STATE: ItwPreferencesState = {};
+
+const reducer = (
+  state: ItwPreferencesState = INITIAL_STATE,
+  action: Action
+): ItwPreferencesState => {
+  switch (action.type) {
+    case getType(itwCloseFeedbackBanner): {
+      return {
+        hideFeedbackBannerUntilDate: addMonths(new Date(), 1).toDateString()
```

Comment:
Is there a specific reason to use `toDateString` and not `toISOString`? I think ISO 8601 representation is more robust. What do you think?

Reply:
Not a specific reason, I think it's shorter and has all the info we need. However, you're right! Prioritizing robustness would be better.
Addressed here: https://github.com/pagopa/io-app/pull/6378/commits/5b787b0868bae6373540d3c8d2c22a2c4bd8b2dc